sys/include/net/gnrc/ipv6.h

```diff
@@ -164,6 +169,11 @@ extern "C" {
 #endif /* DOXYGEN */
 /** @} */
 
+/**
+ * @brief Message queue size to use for the IPv6 thread.
+ */
+#define GNRC_IPV6_MSG_QUEUE_SIZE    (1 << CONFIG_GNRC_IPV6_MSG_QUEUE_SIZE_EXP)
```

Comment:
I am a bit wary though not making this configurable via CFLAGS anymore :-/. Many applications might rely on the fact that it is.

Reply:
Do you suggest deprecating this instead of removing the `#ifdef` now?

Reply:
I don't see a reason outright shutting that door in any case (except for enforcing Kconfig). I'd say, keep the `#ifdef` and if and only if there comes a reason for discouraging the configuration via `CFLAGS`, we can always deprecate it then.

Reply:
I'm not discouraging configuration via `CFLAGS`, I just think that it's clearer for the user to have a single entry-point to configure this parameter (which can still be configured via `CFLAGS`).
Anyway, I re-added the `#ifdef`.